### PR TITLE
fix #79151 linkneverdie.net

### LINKED ISSUE #79151: linkneverdie.net
### Issue URL (Anti Adblock Script)
 [https://linkneverdie.net](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Flinkneverdie.net)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Edge
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion

### PR DESCRIPTION
**Pull request by wout-codes**
#79151

### CODE REVIEW

File: EnglishFilter/sections/foreign.txt [hunk 1]
```diff
@@ -9295,6 +9295,8 @@ brandbuffet.in.th#@#.category-advertorial
 !------------------------------ Vietnam ------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/79151
+@@||static.doubleclick.net/instream/ad_status.js$domain=linkneverdie.net
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/foreign.txt:9299]: to antiadblock.txt please

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/foreign.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/foreign.txt:9299]: This is a Vietnamese Site.

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/foreign.txt:9299]: @wout-codes 

ABPVN filter causes antiadblock too so please add this rule:
`@@||tag.gammaplatform.com^$domain=linkneverdie.net`

and this for share button:
`linkneverdie.net##div[title="Chia sẻ"]`

Comment (wout-codes) [reply EnglishFilter/sections/foreign.txt:9299]: Okay.

**Pull request merged**